walk-in-interview/src/main/webapp/delete-job/script.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * This file is specific to index.html for job-details. 
+ * It renders the fields on the page dynamically,
+ * and send PATCH request to mark the current job post as deleted.
+ */
+
+/**
+ * Import statements are static so its parameters cannot be dynamic.
+ * TODO(issue/22): figure out how to use dynamic imports
+ */
+import {AppStrings} from '../strings.en.js';
+
+import {setErrorMessage} from '../common-functions.js';
+
+const STRINGS = AppStrings['delete-job'];
+const HOMEPAGE_PATH = '../index.html';
+const RESPONSE_ERROR = 'There was an error while deleting the job post. Please try again';
+
+window.onload = () => {
+  renderDeleteButton();
```

Comment:
Just curious, if I revisit this page after deleted, will delete button show again?

Reply:
After deleted, there is no way to access this page with specific jobId